.github/workflows/release.yaml

```diff
@@ -1,127 +1,32 @@
-    steps:
-      - name: Checkout code
-        uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # tag=v3
-      - name: Login to GitHub Container Registry
-        uses: docker/login-action@f4ef78c080cd8ba55a85445d5b36e214a81df20a # v2.1.0
-        with:
-          registry: ghcr.io
-          username: ${{ github.actor }}
-          password: ${{ secrets.GITHUB_TOKEN }}
-      - name: Run Trivy in fs mode to generate SBOM
-        uses: aquasecurity/trivy-action@e5f43133f6e8736992c9f3c1b3296e24b37e17f2 # master
+      - name: "☁️  Checkout repository"
+        uses: actions/checkout@v3
         with:
-          scan-type: 'fs'
-          format: 'spdx-json'
-          output: 'spdx.sbom.json'
-      - name: Install cosign
-        uses: sigstore/cosign-installer@dd6b2e2b610a11fd73dd187a43d57cc1394e35f9 # main
-      - name: Sign image and sbom
-        run: |
-          #!/usr/bin/env bash
-          set -euo pipefail
-          cosign sign -a git_sha=$GITHUB_SHA ${IMAGE_URI_DIGEST} --yes
-          cosign attach sbom --sbom spdx.sbom.json ${IMAGE_URI_DIGEST}
-          cosign sign -a git_sha=$GITHUB_SHA --attachment sbom ${IMAGE_URI_DIGEST} --yes
-        shell: bash
-        env:
-          IMAGE_URI_DIGEST: ${{ needs.build.outputs.image }}@${{ needs.build.outputs.digest }}
-
-  provenance:
-    needs: [build]
-    if: startsWith(github.ref, 'refs/tags/')
-    uses: slsa-framework/slsa-github-generator/.github/workflows/generator_container_slsa3.yml@v1.6.0
-    with:
-      image: ${{ needs.build.outputs.image }}
-      digest: ${{ needs.build.outputs.digest }}
-      registry-username: ${{ github.actor }}
-    secrets:
-      registry-password: ${{ secrets.GITHUB_TOKEN }}
+          fetch-depth: 0
 
-  verify:
-    name: Verify image and provenance
-    runs-on: ubuntu-latest
-    needs: [build, sign, provenance]
-    if: startsWith(github.ref, 'refs/tags/')
-    steps:
-      - name: Login to GitHub Container Registry
-        uses: docker/login-action@f4ef78c080cd8ba55a85445d5b36e214a81df20a # v2.1.0
-        with:
-          registry: ghcr.io
-          username: ${{ github.actor }}
-          password: ${{ secrets.GITHUB_TOKEN }}
-      - name: Install cosign
-        uses: sigstore/cosign-installer@dd6b2e2b610a11fd73dd187a43d57cc1394e35f9 # main
-      - name: Install slsa-verifier
-        uses: slsa-framework/slsa-verifier/actions/installer@c9abffe4d2ab2ffa0b2ea9b2582b84164f390adc # v2.3.0
-      - name: Verify image and provenance
-        run: |
-          #!/usr/bin/env bash
-          set -euo pipefail
-          cosign verify ${IMAGE_URI_DIGEST} \
-            --certificate-oidc-issuer ${GITHUB_ACITONS_OIDC_ISSUER} \
-            --certificate-identity ${COSIGN_KEYLESS_SIGNING_CERT_SUBJECT}
-          slsa-verifier verify-image \
-            --source-uri github.com/${{ github.repository }} ${IMAGE_URI_DIGEST}
-        shell: bash
+      - name: "🚀 Release tag"
+        id: semantic-release
```

Comment:
I _believe_ doing this without the `npm` keys will just cut the github release (which will then trigger the docker builder action). Cc @brandonroberts / @Anush008 to confirm 👀

Reply:
Hmmmm maybe not: 

https://github.com/open-sauced/release/blob/70c6d08f116cb1c28af08b9da20b8c31934623b1/release.config.js#L121-L125

I guess what I really need is just the semantic release stuff and the github release (and non of the npm stuff)

Reply:
Yea ..  the release action assumes you have a package.json also, where it will skip trying to deploy to npm if you have `private: true` in the `package.json`. 

So that would need to be fixed upstream unless you used the semantic release with your own local config.

Reply:
We can update the release upstream to conditionally add the `@semantic-release/npm` plugin only of a value for `NPM_TOKEN` is provided.
https://github.com/open-sauced/release/blob/70c6d08f116cb1c28af08b9da20b8c31934623b1/release.config.js#L122

Reply:
Done in: https://github.com/open-sauced/release/pull/22